tests/components/zeversolar/test_init.py

```diff
@@ -27,34 +26,21 @@ async def test_async_setup_entry_fails(
         pytest.raises(ConfigEntryNotReady),
     ):
         await init.async_setup_entry(hass, config_entry)
-
+        assert config_entry.state is ConfigEntryState.SETUP_RETRY
```

Comment:
Better to unindent here.
```suggestion
    assert config_entry.state is ConfigEntryState.SETUP_RETRY
```